src/steps/fetch-computer-groups/index.ts

```diff
@@ -53,11 +55,17 @@ export function createComputerGroupEntity(
         // So for now we will consider computerGroups as
         // generic Group entities.
         _class: 'Group',
-        parentGroupID: group.parentGroupID,
         cloudType: group.cloudType,
-        amazonSubnetID: group.amazonSubnetID,
         type: group.type,
-        ID: group.ID,
+
+        // stringify to match data model
+        id,
+        amazonSubnetId: group.amazonSubnetID
+          ? createComputerGroupEntityIdentifier(group.amazonSubnetID)
+          : undefined,
+        parentGroupId: group.parentGroupID
+          ? createComputerGroupEntityIdentifier(group.parentGroupID)
+          : undefined,
```

Comment:
How do you feel about making `createComputerGroupEntityIdentifier` handle the check?
```suggestion
        amazonSubnetId: createComputerGroupEntityIdentifier(group.amazonSubnetID),
        parentGroupId: createComputerGroupEntityIdentifier(group.parentGroupID),
```

Reply:
Great suggestion. Fixed.